ibis/backends/spark/ddl.py

```diff
@@ -1,6 +1,6 @@
 from ibis.backends.base_sql import quote_identifier
-from ibis.impala import ddl as impala_ddl
-from ibis.impala.ddl import (  # noqa: F401
+from ibis.backends.impala import ddl as impala_ddl
+from ibis.backends.impala.ddl import (  # noqa: F401
```

Comment:
This seems to be the problem. We are still importing impala from spark. Whatever it's used in Spark should be moved to `base_sql`, and not import impala at all. Otherwise spark will load the impala backend, and will fail because it's dependencies won't be installed.

This should be fixed in a separate PR. Do you want to work on it? I can work on it if you prefer.

Reply:
submitted a new PR #2513